aiida/cmdline/params/options/commands/computer.py

```diff
@@ -117,6 +117,16 @@ def should_call_default_mpiprocs_per_machine(ctx):  # pylint: disable=invalid-na
     help='The default amount of RAM (kB) that should be allocated per machine (node), if not otherwise specified.'
 )
 
+USE_DOUBLE_QUOTES = OverridableOption(
+    '--use-double-quotes/--not-use-double-quotes',
+    is_eager=False,
```

Comment:
This is the default and is not necessary
```suggestion
    is_eager=False,
```